src/cursor.ts

```diff
@@ -330,6 +345,12 @@ export function defaultDecodeCursor<
     if (field[0] !== expectedName) {
       throw new Error("Unexpected field name");
     }
+
+    const [_fieldName, value] = field;
+    if (value === "null") {
+      // @ts-ignore
+      field[1] = null;
+    }
```

Comment:
Need to fix this. But `field` is typed as `[string, string]` directly from the URLSearchParams. But basically, if the cursor value is `"null"`, we want to convert it back to actual `null`